src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -409,7 +410,7 @@ public void mergeAlignment(final File referenceFasta) {
                         final SAMRecord matePrimary = isRead1 ? r2Primary : r1Primary;
 
                         for (final SAMRecord supp : supplementals) {
-                            final SAMRecord out = clone(sourceRec);
+                            final SAMRecord out = sourceRec.deepCopy();
```

Comment:
hmmm. I don't think that we should make that assumption. I don't think that supplementary alignments need to be on the same strand as the primary. it doesn't even make sence since they can be on different contigs...IMHO, the right way to avoid deep-copying all the supplementals would be to modify reverseComplement() (unfortunately since it it in htsjdk) and provide it with an optional boolean flag that will make new arrays of bases and qualities instead of reversing in place. this will mean that we don't actually do a deep copy, but that we protect other copies of the bases and qualities from modification.